markdown/3.x/en/ff-sortbox.md

```diff
@@ -3,15 +3,32 @@ You can choose from two different kinds of sortboxes. The `ff-sortbox-select` is
 The `ff-sortbox` is a custom-built dropdown element. Both can be used with or without customization.
 
 ## Native HTML select
-The code below will display a native HTML `select`. The template specified through the `data-template` attribute is 
-optional. It will default to `<option>{{description}}</option>` if not present.
+The code below will display a native HTML `select`. The `select`-element as well as the `option`-element are optional. It will default to `<select><option>{{description}}</option></select>` if not present.
 
 ```html
 <ff-sortbox-select>
-    <option data-template>Sort by: {{description}}</option>
```

Comment:
Examples of all possible setups would be appreciated by readers I suppose.
We could simply list three code blocks without much explanation. Just some one-liners:

Minimal setup:
```
<ff-sortbox-select></ff-sortbox-select>
```

Specifying only the `option` element:
```
<ff-sortbox-select>
  <option>{{description}}</option>
</ff-sortbox-select>
```

Specifying `select` and `option` elements:
```
<ff-sortbox-select>
    <select class="example-class">
        <option>Sort by: {{description}}</option>
    </select>
</ff-sortbox-select>
```

The "rendered HTML" section can stay the same, just mentioning which of the examples it is referring to.